hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/protocol/ClientProtocol.java

```diff
@@ -997,6 +997,10 @@ void setBucketQuota(String volumeName, String bucketName,
   void setReplicationConfig(String volumeName, String bucketName,
       ReplicationConfig replicationConfig) throws IOException;
 
+  @Deprecated
+  void setEncryptionKey(String volumeName, String bucketName,
+                        String bekName) throws IOException;
```

Comment:
Please add javadoc and explain why it's deprecated, so readers don't have to dig up the PR.

Reply:
Thank you @adoroszlai, I have added a javadoc explaining why the functionality has been deprecated.